index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link rel=" stylesheet" href="style/style.css">
+    <title>Portifolio</title>
+</head>
+
+<body>
+    <div class="container">
+        <header>            
+            <div class="toolbar">
+                <div class="logo">
+                    <h2>My Logo</h2>
+                </div>
+                <div class="menu"><img src="assets/Union.png" alt="Menu"></div>
+            </div>
+        </header>
+        <div class="content">
+            <h1>I’m Guilherme Glad to see you!</h1>
+            <div>
+                <p>
+                    I’m a software developer! I can help you build a product , feature or website Look through some of
+                    my work and experience! If you like what you see and have a project you need coded, don’t hestiate
+                    to contact me.
+                </p>
+            </div>
+            <div>
+                <h2><a href="#">Let's Connect</a></h2>
+            </div>
+            <div class="social">
+                <a href="#"><img src="assets/Twitter.png" alt="Twitter"></a>
+                <a href="#"><img src="assets/Linkedin.png" alt="Linkedin"></a>
+                <a href="#"><img src="assets/M.png" alt="M"></a>
+                <a href="#"><img src="assets/Git.png" alt="GitHub"></a>
+                <a href="#"><img src="assets/Hand.png" alt="Hand"></a>
+            </div>
```

Comment:
- [ ] The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag), please change it

Reply:
:+1: Issue addressed successfully :heavy_check_mark: